### PR TITLE
Split HttpListener and Kestrel implementation

### PR DESCRIPTION
**Pull request by sanych-sun**
@phnx47 Could you please take a look, update build script to create a new package and merge it. Please note this is intermediate version, just to check if it helps to solve ACL problem.

### CODE REVIEW

File: src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>MetricServer for the Prometheus.Client</Description>
+    <Copyright>2019 © Sergey Kuznetsov</Copyright>
+    <AssemblyTitle>Prometheus.Client.MetricServer</AssemblyTitle>
+    <VersionPrefix>3.0.2</VersionPrefix>
+    <Authors>Sergey Kuznetsov</Authors>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
```
Comment (phnx47) [inline src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj]: We should add neststandard2.0 and netcoreapp2.2 for Kestrel

File: src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <PackageTags>prometheus;metrics</PackageTags>
+    <PackageIconUrl>https://image.ibb.co/k4Sc0k/prometheus.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/PrometheusClientNet/Prometheus.Client.MetricServer</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/PrometheusClientNet/Prometheus.Client.MetricServer</RepositoryUrl>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../../Prometheus.Client.MetricServer.snk</AssemblyOriginatorKeyFile>
```
Comment (phnx47) [inline src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj]: We should generate new *.snk

File: src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj (hunk 1 above)
Comment (sanych-sun) [reply src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj]: netstandard2.0 was added.

File: src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj (hunk 2 above)
Comment (sanych-sun) [reply src/Prometheus.Client.MetricServer.Kestrel/Prometheus.Client.MetricServer.Kestrel.csproj]: We might, but what for? Several libraries can share key.

**Pull request merged**